test/runtime/Policies.go

```diff
@@ -631,6 +647,37 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 		connectivityTest(allRequests, helpers.App2, helpers.Httpd1, true)
 	})
 
+	It("L3-Dependent L7 Egress", func() {
+		_, err := vm.PolicyImportAndWait(vm.GetFullPath(policiesL3DependentL7EgressJSON), helpers.HelperTimeout)
+		Expect(err).Should(BeNil(), "unable to import %s", policiesL3DependentL7EgressJSON)
+
+		endpointIDS, err := vm.GetEndpointsIds()
+		Expect(err).To(BeNil(), "Unable to get IDs of endpoints")
+
+		app3EndpointID, exists := endpointIDS[helpers.App3]
+		Expect(exists).To(BeTrue(), "Expected endpoint ID to exist for %s", helpers.App3)
+
+		connectivityTest(httpRequestsPublic, helpers.App3, helpers.Httpd1, true)
+		checkProxyStatistics(app3EndpointID, 2, 2, 0, 2, 2)
+
+		connectivityTest(httpRequestsPrivate, helpers.App3, helpers.Httpd1, false)
+		checkProxyStatistics(app3EndpointID, 2, 4, 2, 2, 2)
+
+		connectivityTest(httpRequestsPublic, helpers.App3, helpers.Httpd2, true)
+		checkProxyStatistics(app3EndpointID, 4, 6, 2, 4, 4)
+
+		connectivityTest(httpRequestsPrivate, helpers.App3, helpers.Httpd2, true)
+		checkProxyStatistics(app3EndpointID, 6, 8, 2, 6, 6)
+
+		By("Disabling all the policies. All should work")
+
+		status := vm.PolicyDelAll()
+		status.ExpectSuccess()
+		vm.WaitEndpointsReady()
```

Comment:
no error&assert here?

Reply:
Removed this section since `AfterEach` here deletes all policies.